### PR TITLE
Disabled Rubocops, all available listed in config

### PR DESCRIPTION
**Pull request by kotp**
re: #443

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,30 +1,1922 @@
...
+  StyleGuide: https://github.com/bbatsov/ruby-style-guide#hash-key
+  Enabled: false
+
+Style/Documentation:
+  Description: Document classes and non-namespace modules.
+  Enabled: false
   Exclude:
-    - 'exericises/**/*_test.rb'
+  - "/home/victor/Projects/Exercism/!exercism_org/xruby/spec/**/*"
+  - "/home/victor/Projects/Exercism/!exercism_org/xruby/test/**/*"
```
Comment (Insti) [inline .rubocop.yml]: Probably shouldn't be using absolute paths.

Comment (kotp) [inline .rubocop.yml]: Generated... yeah, gotta fix that.  I noticed it, haven't updated it.  Coming in.

Comment (kotp) [inline .rubocop.yml]: How else are you supposed to make sure my style is good? lol.  Fixed it.

**Comment by kotp** [comment]
Considering how to update the [README](https://github.com/exercism/xruby/blob/master/README.md) to keep it in tune with the changes... ideas?

**Pull request merged**